src/Illuminate/View/Concerns/ManagesLoops.php

```diff
@@ -73,7 +73,7 @@ public function popLoop()
      */
     public function getLastLoop()
     {
-        return $last = Arr::last($this->loopsStack) ? (object) $last : null;
+        return ($last = Arr::last($this->loopsStack)) ? (object) $last : null;
```

Comment:
We enclose the declaration of `$last` inside brackets so that we can use it in the condition.